go/felix/intdataplane/ipip_mgr_test.go

```diff
@@ -0,0 +1,444 @@
+// Copyright (c) 2017 Tigera, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package intdataplane
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"errors"
+	"fmt"
+	log "github.com/Sirupsen/logrus"
+	"github.com/projectcalico/felix/go/felix/proto"
+	"github.com/projectcalico/felix/go/felix/set"
+	"github.com/vishvananda/netlink"
+	"net"
+)
+
+var (
+	notFound    = errors.New("not found")
+	mockFailure = errors.New("mock failure")
+)
+
+var _ = Describe("IpipMgr (tunnel configuration)", func() {
+	var (
+		ipipMgr   *ipipManager
+		ipSets    *mockIPSets
+		dataplane *mockIPIPDataplane
+	)
+
+	ip, _, err := net.ParseCIDR("10.0.0.1/32")
+	if err != nil {
+		panic("Failed to parse test IP")
+	}
+	_, ipNet2, err := net.ParseCIDR("10.0.0.2/32")
+	if err != nil {
+		panic("Failed to parse test IP")
+	}
```

Comment:
I'm thinking of these as constants, it's simply annoying that the `err` requires a bit of extra handling.